sys/src/9/amd64/vq9p.c

```diff
@@ -0,0 +1,216 @@
+/*
```

Comment:
It is probably intel-specific as is all virtio (does Linux on ARM provide identical functionality in terms of virtualization, and is it in the same wide use as on amd64)?